### PR TITLE
Feature/your great

### PR DESCRIPTION
**Pull request by georgehipp**
## Synopsis of Changes

  - Added new Journey of  Your Career section
  - Updated build script
  
## Proposed Changes

- Added new Journey of  Your Career section to People Chapter with You're Great article adjusted for non VMLY&R readers
  - Updated build script to work properly with new citeproc and to stop error from no pdf file
  
## Steps to Test
  
  - Pull this Branch
    - git pull
    - git checkout [This Branch]
    - sh build.sh
    - Validate Build Worked

### CODE REVIEW

File: en/06_people/05_journey_of_your_career.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+## Journey of your Career
+
+### Your Great!
```
Comment (lyndseypadget) [inline en/06_people/05_journey_of_your_career.md:3]: You're ?   (Was this intentional?)

File: en/06_people/05_journey_of_your_career.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+You are pretty sure you are good at what you do, some day's you might even say you are amazing at your job! How do you make sure others know how good you are - I don't mean in the periphery of their mind "they do good work" sort of way, but have your name at the tip of their tongue when new opportunities come up? Is not easy, especially when you are surrounded by others doing great work at the same time; here are some ideas to help you out.
```
Comment (lyndseypadget) [inline en/06_people/05_journey_of_your_career.md:5]: some days (no apostrophe)

File: en/06_people/05_journey_of_your_career.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+I like lists of three's, short enough to remember and long enough to get some good direction.
```
Comment (lyndseypadget) [inline en/06_people/05_journey_of_your_career.md:7]: threes

File: en/06_people/05_journey_of_your_career.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+Try creating a few of these statements, feel them out, see how they fit you.
+
+Some things to consider when making these:
+
+* Ensure you enjoy working in your target area, you may be spending a lot of time there
+* Check to see if there is work where you are aiming!
+    * Wanting to be the best ALGOL developer may not make sense
+* Make sure it aligns with your career ladder somehow
+* Your organization Leveling Guide
```
Comment (lyndseypadget) [inline en/06_people/05_journey_of_your_career.md:29]: The other bullets are a complete sentence (a direction or command) but this one isn't.

File: en/06_people/05_journey_of_your_career.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+With a strategy and evidence, you are ready to share! How do you get traction and figure out who to share what with?
+
+Start with your Pod Lead or Direct Manager, they have the largest ability to affect your work and the most interest in ensuring everyone knows how good you are. They can be your partner is refining your brand and getting the word out. Hint, this is all great content for your 1on1's!
+
+Engage and communicate with the Tech Lead on your project. They should know and be aware of the great things you are doing! They may or may not be in your organization or leadership, but it is still important they know you! Make sure they understand where you are knowledgeable.
+
+The Client Engagement person on your project should know you are someone they can call on for help and to answer questions in the area you are an expert in.
+
+Create influence everywhere you can!
```
Comment (lyndseypadget) [inline en/06_people/05_journey_of_your_career.md:46]: The purpose of this section is to pick an audience, and this bit of advice seems to contradict that.

File: en/06_people/05_journey_of_your_career.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+Here are some examples if your audience is Technical Leadership:
+
+* Start writing about your expertise here in the Technology Blog
```
Comment (lyndseypadget) [inline en/06_people/05_journey_of_your_career.md:50]: Copy pasta... we're not in the Blog, anymore.

File: en/06_people/05_journey_of_your_career.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+* Answer questions in Teams, Slack, Discord (No Stupid Questions Club, Technology General, etc.)
```
Comment (lyndseypadget) [inline en/06_people/05_journey_of_your_career.md:51]: The part in parentheses are too specific for a public document.

File: en/06_people/05_journey_of_your_career.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+* Spin up small group meetings showing others what you know
+* Get involved in the community surrounding the space you are going after
+* Check out some open source: https://github.com/explore
+* Join Stack Overflow Collectives: https://stackoverflow.com/collectives
+* Find CoE's and groups to join in your organization
+* If there is no Meetup, make one! https://www.meetup.com/start/organizing
+* Volunteer to help in anything related to your area
```
Comment (lyndseypadget) [inline en/06_people/05_journey_of_your_career.md:58]: Lines 54-58 seem like sub-bullets of line 53.

**Review by lyndseypadget — APPROVED** [review]
As we've discussed, I don't necessarily think this content belongs in this publication.  However, @davidmitchell thinks it does (or _could_)... and he's the boss, applesauce. 😉

So, I've approved this and included review comments which are most grammatical or structural in nature.

Additional considerations:
- Personally, the exclamation point ratio seems high in this content.  That might just be me, but we should check that the tone of incoming content matches the tone of the publication thus far.
- To that end, the purpose of the publication is to help clients understand what JDD is, why we do it, and what it entails.  (This brings my review full circle...) This content seems more focused on an individual technologist's professional branding.  If the idea could be reworked through the lens of B2B or B2C, would it be a better fit?  I think yes. 

Perhaps @sykesjs would like to weigh in.

**Comment by davidmitchell** [comment]
Think there is some good thinking here. Lyndsey has provided good feedback. Unfortunately, George isn't at VML anymore so I'm going to close this.

**Pull request closed without merging**